### ISSUE TITLE
No visual changes available on focus state/pressed for some buttons on Android

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#655]
Scenario to reproduce

Samsung Galaxy J3 Prime
Android version 7.0
App version: 1.0 (46)

For the buttons such as “Enter number”, “Next” there is no focus state indicator when touched. Also, when pressed to activate them no visual change for users to know that they pressed the buttons.

Expected result
Users should be able to see the buttons have focus and they are pressed from the visual change.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

Screenshots
![button](https://user-images.githubusercontent.com/12256067/87319066-24b48d00-c4f7-11ea-832f-8b7e2f16f05a.gif)

**Comment by JuliannaR** [comment]
@nickmly is this related to the touchhighlight fix for contrast?

**Comment by nickmly** [comment]
@JuliannaR I am unfamiliar with this issue, can you clarify?

**Comment by JuliannaR** [comment]
@nickmly sorry, I mis-remembered you being on this. When you self assigned I thought you worked on the previous issue.
https://github.com/cds-snc/covid-shield-mobile/issues/600 is a closed issue that might have impacted this one.

**Comment by nickmly** [comment]
@JuliannaR I see. This issue should not impact the fix for color contrast, it relates to the 'ripple' animation on Android that is not playing for some reason when focusing a button
